test-nifikop-operator/deploy/zk.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Just to confirm, it seems that nifi needs zk to run?

Reply:
Yes, I add this script to manually deploy zk before start the operator.